src/components/elements/SemanticIcons.tsx

```diff
@@ -0,0 +1,4 @@
+import Home from '@mui/icons-material/Home';
+import Person from '@mui/icons-material/Person';
+
+export { Person as ClientIcon, Home as EnrollmentIcon };
```

Comment:
suggest we could extract icons from other components to make it easier to stay consistent.

Reply:
THANK YOU I have been wanting it to be like this for so long but never got around to it